gtsfm/loader/hilti_loader.py

```diff
@@ -56,18 +58,27 @@ def __init__(
         base_folder: str,
         max_length: Optional[int] = None,
         max_resolution: int = 1080,
+        subsample: int = 1,
+        old_style: bool = False,
```

Comment:
can we remove old_style?

Reply:
I'll handle it as a different PR. Too many changes as-it-is in the current PR.

Reply:
can you add a TODO?

Reply:
Added a TODO and created an issue.